app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def defualt
```

Comment:
__[Misspell]__

"defualt" is a misspelling of "default"


<!-- Comment by [Sider](https://sider.review) -->